Panic: index out of range when -c/--config has no argument

Running wuzz with a trailing `-c` or `--config` flag and no following argument crashes with a panic:

```
$ wuzz -c
panic: runtime error: index out of range [2] with length 2

goroutine 1 [running]:
main.main()
	wuzz.go:1869
```

In `main()`, the `-c`/`--config` case reads `os.Args[i+1]` without a bounds check, unlike the other value flags in `ParseArgs` which are all bounds-checked. When the flag is the last argument, indexing past the slice panics.

Expected: exit with a clear error message, as with other invalid usage.

--- a/wuzz.go
+++ b/wuzz.go
@@ -1866,6 +1866,9 @@ func main() {
 			fmt.Printf("wuzz %v\n", VERSION)
 			return
 		case "-c", "--config":
+			if i == len(os.Args)-1 {
+				log.Fatal("No config file specified")
+			}
 			configPath = os.Args[i+1]
 			args = append(os.Args[:i], os.Args[i+2:]...)
 			if _, err := os.Stat(configPath); os.IsNotExist(err) {

